source/ui/Components/editor/Editor.qml

```diff
@@ -635,19 +732,19 @@ ScrollView {
                         property alias color: breakpointIcon.color
                         width: breakpointTrigger.width
                         height: textArea.cursorRectangle.height;
+
                         Component.onCompleted: {
                             editor.setBreakpoint(line + 1);
                         }
 
                         Rectangle {
                             id: breakpointIcon
-                            height: Math.min(parent.height, errorBar.width) * 0.8
+                            height: textArea.cursorRectangle.height -1
                             width: height
                             anchors.verticalCenter: parent.verticalCenter
-                            anchors.left: parent.left
-                            anchors.leftMargin: height*0.15

```

Comment:
Why did you change the height and anchors of this? It seems to work worse for me now, the breakpoints are a bit too big and sometimes clip with the left edge. The height with `Math.min` was to prevent them from being bigger then the height or width of the breakpoint bar.